File: Lv1/제일 작은 수 제거하기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    if(arr.size()==1)
        answer.push_back(-1);
    else{
        int min_idx=0;
        for(int i=1; i<arr.size(); i++){
            if(arr[min_idx]>arr[i])
                min_idx=i;
        }
        arr.erase(arr.begin()+min_idx);
       answer=arr;
    }
    return answer;
}

File: Python/Programmers/Lv1/제일 작은 수 제거하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12935?language=python3

def solution(arr):
    answer=arr
    min=answer[0]
    for i in answer:
        if min > i:
            min=i
    answer.remove(min)
    if len(answer)==0:
        answer.append(-1)
    return answer